ICD10GM/manual_work/pre_load_stage.sql

```diff
@@ -1,3 +1,7 @@
+-- delete from concept_manual table "dead" concepts
```

Comment:
only this query is actualy run, other queries you shouldn't run as concepts are already there